gst/nnstreamer/tensor_common.c

```diff
@@ -803,7 +803,8 @@ gst_tensors_config_validate (const GstTensorsConfig * config)
   }
 
   /* cannot check tensor info when tensor is flexible */
-  if (gst_tensors_config_is_flexible (config)) {
+  if (gst_tensors_config_is_flexible (config) ||
```

Comment:
Discuss: flex or sparse > not static

Reply:
Great thanks! I changed this line to `!gst..._is_static `